dsl/pace/dsl/stencil.py

```diff
@@ -103,17 +103,16 @@ def _get_backend_opts(
         return backend_opts
 
     def stencil_kwargs(self, skip_passes: Iterable[str] = ()):
+        # NOTE (jdahm): Temporary replace call until Jenkins is updated
         kwargs = {
-            "backend": self.backend,
+            "backend": self.backend.replace("gtc:", ""),
             "rebuild": self.rebuild,
             **self.backend_opts,
         }
         if not self.is_gpu_backend:
             kwargs.pop("device_sync", None)
         # Note: this assure the backward compatibility between v36 and v37
-        if self.backend.startswith("gtc") and (
-            "skip_passes" in kwargs or skip_passes is not None
-        ):
+        if "skip_passes" in kwargs or skip_passes is not None:
```

Comment:
I know your refactor hasn't changed this code, but is this code wrong? skip_passes can never be `None`.

Reply:
Removed. Nice catch!